API/Controllers/OfferController.cs

```diff
@@ -52,5 +52,13 @@ public async Task<IActionResult> GetUserOffersAsync([FromQuery] PaginationFilter
             var userId = _userService.GetCurrentUserNameIdentifier(User);
             return Ok(await _offerService.GetUsersOffersAsync(userId, paginationFilter));
         }
+
+        [HttpGet("trip-offers")]
+        [AuthorizeByRole(IdentityRoleNames.Logist)]
+        public async Task<IActionResult> GetOfferByTrip(int tripId)
+        {
+            var offers = await _offerService.GetOfferByTripAsync(tripId);
```

Comment:
We will return a list of trip`s offers. So I think it's better to name this method and endpoint, like GetOffersByTripAsync, in the plural.
This is not critical